Logic/MovieLogic.cs

```diff
@@ -16,44 +17,586 @@ public MovieLogic(RepoLogic repo)
         {
             _repo = repo;
         }
-        public async Task<bool> CreateMovie(string movieId)
+
+        public async Task<MovieDTO> GetMovie(string movieId)
```

Comment:
comments in this whole file

Reply:
The methods without comments are implementations of an interface that is commented